### ISSUE TITLE
Latest git snapshot failed to compile on aarch64

### ISSUE THREAD

**Issue by tim77** [libretro/pcsx_rearmed#651]
## Description

Latest 3558dd33fb337a29fb6c57a5e88a473153e6202e won't compile anymore on aarch64 arch. 

## When did the behavior start?

Latest know build which compiles successfully is f5b7bb83abb50a2028cd8fb16ebf75bd3df8cba2 dated 2021-10-04.

## Your device/OS/platform/architecture

Fedora Linux 36+, aarch64.

## Logs (enable file logging and set log levels to DEBUG for core and frontend)

[Full build log](https://kojipkgs.fedoraproject.org//work/tasks/8112/86228112/build.log).

**Comment by tim77** [comment]
@pcercuei many thanks! Compiled succesfully with `ALLOW_LIGHTREC_ON_ARM=1`.

**Comment by notaz** [comment]
I'd still recommend compiling with DYNAREC=ari64 instead to get better compatibility and performance from some game-specific hacks.

**Comment by pcercuei** [comment]
@notaz shouldn't it be the default then? Looks like Lightrec is still the default on Linux/Aarch64.

**Comment by notaz** [comment]
Right, just pushed an attempt to this. The makefiles are messy and feel fragile, so I'm always reluctant to change them.
